lesson_2/Lesson_2.1.ipynb

```diff
@@ -0,0 +1,605 @@
+{
```

Comment:
<p>Why not move this to section 3 where you are talking about PDFs anyway?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/AlexAndorra/notebooks-experiments/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
<p>Not opposed just talking this through</p><p> </p><p>It underscores how distributions are defined mathematically but well be usnig them with code. I can put this in both places though, I dont think theres any harm in repetition, especially in important topics to reinforce them. Students might miss it the first time but when they see it again theyll start picking up its important</p><p> </p><p>Thoughts?</p><p> </p>